redfish-exporter/redfish_utils.go

```diff
@@ -268,3 +269,19 @@ func getServerInfo(redfishServers []RedfishServer, serverIP string) RedfishServe
 	}
 	return RedfishServer{}
 }
+
+func getServerInfoWithRawIP(redfishServers []RedfishServer, IP string) RedfishServer {
```

Comment:
I had to do some similar stuff for Prometheus metrics here:
https://github.com/nod-ai/ADA/blob/main/redfish-exporter/main.go#L136
Perhaps we can move it to this module and reuse it. otherwise, you could follow the next comment.

Reply:
Thank you. Re-using `extractHost`.